packages/core/src/etherscan.ts

```diff
@@ -35,7 +35,6 @@ export const getChainConfig = (chainId: number): ChainConfig => {
   const chainConfig = [
     // custom chains has higher precedence than builtin chains
     ...[...customChains].reverse(), // the last entry has higher precedence
-    ...builtinChains,
```

Comment:
I dropped the use of `buildinChains` because we always rely on the block explorer info we define ourselves in the `SPHINX_NETWORKS` array.